app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -28,6 +28,7 @@ def index
     param :start_date, String, :desc => N_("erratum: start date (YYYY-MM-DD)")
     param :end_date, String, :desc => N_("erratum: end date (YYYY-MM-DD)")
     param :types, Array, :desc => N_("erratum: types (enhancement, bugfix, security)")
+    param :use_issued_date, :bool, :desc => N_("erratum: search using the 'Issued On' date and not 'Updated On', useful only if dates are used")
```

Comment:
If this was the default previous behavior, and the goal is to maintain backwards compatibility, shouldn't this be defaulted to true?